scripts/gateway-docker-upgrade.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-set -euo pipefail
```

Comment:
This was causing the script to exit `1` without printing a failure message due to line 7 failing.